src/ContextMenu.tsx

```diff
@@ -11,6 +11,10 @@ import { ITextProps, Text } from './Text';
 import { styled } from './utils';
 
 // TODO: expose SubMenu component
+// TODO: allow custom renderMenu
+// TODO: allow custom renderMenuItem?
+// TODO: add icon support to menu items
+// TODO: Context Menu should probably leverage/use MENU
```

Comment:
Thought here is that contextMenu should use menu, we want them to look the same/have similar functionality except one opens on right click and the other shows on click/hover